build.sbt

```diff
@@ -115,10 +45,9 @@ lazy val nodeJs = {
 
 lazy val api = crossProject(JVMPlatform, JSPlatform)
   .in(file("api"))
-  .enablePlugins(PlayLibrary, Playdoc)
+  .enablePlugins(Common, Playdoc, Omnidoc, PublishLibrary)
```

Comment:
It's counterintuitive to have the `Playdoc` plugin on the `api` project and the `docs/` folder on the root folder.

It works because `Playdoc` uses a hardcoded path and piggybacks the `publish` task from `api` but `Playdoc` is completely unrelated to `api`. 

^^^ Not a comment, just a realization. 😒

Reply:
I'm fine to remove it if it doesn't make sense. I keep it because it was explicitly added to it, as you can see from the diff.